### PR TITLE
feat: [search] call change event when clear button is clicked

### PR DESCRIPTION
**Pull request by eljefe223**
# Pull Request

## 📖 Description

Calls `handleChange()` in `handleClearInput()` 

### 🎫 Issues

Closes: #6228 

### General

- [x] I have included a change request file using `$ yarn change`
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by EisenbergEffect — APPROVED** [review]

File: change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (scomea) [inline change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json:2]: should "type" and "dependentChangeType" be "prerelease"?  @chrisdholt

File: change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "comment": "chore: call change when clear button is clicked",
```
Comment (scomea) [inline change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json:3]: Isn't "chore" for something that doesn't affect released packages?  Seems like a "feat" or "fix" to me.

**Review by scomea — APPROVED** [review]

File: change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json (hunk 1 above)
Comment (nicholasrice) [inline change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "packageName": "@microsoft/fast-foundation",
+  "email": "jes@microsoft.com",
+  "dependentChangeType": "patch"
```
[the remark this answers is not in the document]
Comment (nicholasrice) [reply change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json:6]: Oops this one should be changed too

File: change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json (hunk 2 above)
Comment (chrisdholt) [reply change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json:3]:
```suggestion
  "comment": "call change when search component clear button is clicked",
```

Comment (chrisdholt) [reply change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json:3]: I think we can drop that and make the actual change file a bit more clear

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**